### DISCUSSION TITLE
An option to display certification as pdf/jpeg as it's done in LinkedIn skill/certification section

### DISCUSSION THREAD

**Discussion by VaishnavJois** [EddieHubCommunity/LinkFree#8923] [Ideas]
A preview option for certifications achieved that we get as pdf/jpeg that can be displayed in milestones tab or a new section dedicated for certifications/skill badge display

**Comment by waellerbe** [comment]
> A preview option for certifications achieved that we get as pdf/jpeg that can be displayed in milestones tab or a new section dedicated for certifications/skill badge display

Hello @VaishnavJois : In keeping with your idea, I wonder whether we can borrow concepts from the `badges` that we display within our GitHub profiles.

**Reply by VaishnavJois** [reply]
Yes we can, but a thing to consider here:
GitHub badges can't be spoofed as it gets rewarded automatically based on your achievement. But a certification/skill badge from external site should require a proof to legitimize; say a verify button as it's available in Credly or we should have credential URL mandatory
And also not just badges, we would require an option to post PDF too as most courses do not offer badges

**Comment by vedas-dixit** [comment]


**Reply by eddiejaoude** [reply]
> Hello I'm new to this whole open source contribution! I completed the local development with docker setup and ran into a little problem, It's running perfectly fine but when I Modify the code locally it won't reflect on the server(localhost). I'm very confused at this point!

@vedas-dixit please start a new discussion or ask in EddieHub Discord, as this is off topic.not related to this discussion

**Comment by eddiejaoude** [comment]
> A preview option for certifications achieved that we get as pdf/jpeg that can be displayed in milestones tab or a new section dedicated for certifications/skill badge display

@VaishnavJois we have milestones, these could include your achievements

**Reply by VaishnavJois** [reply]
Yes @eddiejaoude . I'm using it already!, but an option to showcase more detail than just texts could add more authenticity is what I meant.
Displaying it as an embedded PDF than using URL section within Milestone section to pop-out to a new tab
